src/org/rust/cargo/icons/CargoIcons.kt

```diff
@@ -0,0 +1,8 @@
+package org.rust.cargo.icons
+
+import com.intellij.openapi.util.IconLoader
+
+public object CargoIcons {
+    public val ICON = IconLoader.getIcon("/org/rust/lang/icons/cargo.png")
+    public val LOCK_ICON = IconLoader.getIcon("/org/rust/lang/icons/cargo-lock.png")
```

Comment:
I'll do that but unless I'm blind there doesn't seem to be an option for the IntelliJ formatter to do this automatically?!